src/transformStateWithClones.js

```diff
@@ -7,7 +7,38 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const statesArray = [];
```

Comment:
that is not necessary to explicitly write what datatype it is
You can do it just like that:
```suggestion
  const states = [];
```
or 
```suggestion
  const stateVersions = [];
```